### PR TITLE
Text improvements

### LINKED ISSUE #210: Add 'AntennaPod on the Linux desktop, iOS and Windows' page
**Short description**: Add a page titled 'AntennaPod on the Linux desktop, iOS and Windows'. This page can contain a message 'AP is and will (for the foreseeing future) focus on Android' as well as our suggestions for great open source podcast apps/applications.
* Linux: https://vocalproject.net/, https://apps.kde.org/kasts/
* Windows: ??
* iOS: ??

**Location**: https://antennapod.org/documentation/general

**Why have this**: Quite a few people ask on Mastodon about linux desktop apps and "antennapod ios" (4th - 976 impressions through this search term the past 12 months) & "antennapod windows" (8th - 518) are relatively popular search terms to get to our website.

**More info**:

### PR DESCRIPTION
**Pull request by keunes**
Closes  #210
Closes #268

### CODE REVIEW

File: _i18n/en/documentation/general/other-platforms.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+However attractive the title may sound, AntennaPod is not and will not be available on Windows, iOS or the Linux desktop. The app is made by a small team of volunteers, and making the app for these other platforms would be a lot of work.
+
+There are (open source) alternatives, however, which we would like to point you to:
+App/Application|Windows|iOS|Linux
+---|---|---|---
+[Kasts](https://apps.kde.org/kasts/)|No|No|Yes
+[Vocal](https://vocalproject.net/)|No|No|Yes
+[gPodder](https://gpodder.github.io/)|Yes|Yes|Yes
+[Pocket Casts](https://pocketcasts.com/)|No|Yes|No
```
Comment (ByteHamster) [inline _i18n/en/documentation/general/other-platforms.md:9]: Not sure if I would mention pocket casts. As far as I know it's a centralized podcast app with a closed-source server (which is where all the interesting and maybe privacy sensitive stuff happens)

File: _i18n/en/documentation/general/other-platforms.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+[Podverse](https://podverse.fm/about)|No|Yes|No
```
Comment (ByteHamster) [inline _i18n/en/documentation/general/other-platforms.md:10]: Maybe a column "can sync with AntennaPod"?

File: _i18n/en/documentation/general/f-droid.md [hunk 3]
```diff
@@ -1,10 +1,18 @@
-After a new version of AntennaPod gets released, we often are asked when it will be available on F-Droid. We can't give an exact answer — it usually takes a few days.
+Did you see that there's a new AntennaPod release and wonder:
+* _why isn't the latest version not on F-Droid yet?_
+* _when will the new release be available on F-Droid?_
 
-Getting a new version on F-Droid involves the following steps:
+If the new version is in beta, it is _only_ available through Google Play ((here is why)[/documentation/general/beta]). When the new version is ready for the public, we upload it to Google Play and tag the release on GitHub. F-Droid then has to find out about this release, build it and make it available. All these steps may take some time.
```
Comment (ByteHamster) [inline _i18n/en/documentation/general/f-droid.md:5]:
```suggestion
If the new version is in beta, it is _only_ available through Google Play ((here is why)[/documentation/general/beta]). When the new version is ready for the public, we upload it to Google Play. A few days later when we are sure that there are no critical bugs, we tag the release on GitHub for F-Droid to find. F-Droid then has to find out about this release, build it and make it available. All these steps may take some time.
```

File: _i18n/en/documentation/bugs-first-aid/database-error.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
+In a very old AntennaPod version, there were some issues with the database with all information (episodes, listening states, etc). While the cause is long fixed, your database might have been damaged (even when the app was working without issues). If a recent AntennaPod version makes changes to the damaged parts of the database, these damages may suddenly become a problem. At that point AntennaPod can no longer work with the broken database and has to start over with an empty one.
+
+We realise it is very frustrating when this happens and are really sorry if you were affected.
```
Comment (ByteHamster) [inline _i18n/en/documentation/bugs-first-aid/database-error.md:5]:
```suggestion
We realize it is very frustrating when this happens and are really sorry if you were affected.
```

File: _i18n/en/documentation/general/other-platforms.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+However attractive the title may sound, AntennaPod is not and will not be available on Windows, iOS or the Linux desktop. The app is made by a small team of volunteers, and making the app for these other platforms would be a lot of work.
+
+There are (open source) alternatives, however, which we would like to point you to:
+App/Application|Windows|iOS|Linux
```
Comment (ByteHamster) [inline _i18n/en/documentation/general/other-platforms.md:4]: I would just write "app". Windows calls it "app" nowadays as well, I think

File: _i18n/en/documentation/general/other-platforms.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+---|---|---|---
+[Kasts](https://apps.kde.org/kasts/)|No|No|Yes
```
Comment (ByteHamster) [inline _i18n/en/documentation/general/other-platforms.md:6]: Yes/no tables are quite hard to read at a glance IMO. How about leaving it empty for "no" and adding a check mark symbol (✓) for "yes"? Then we can also add do-not-translate around the table

File: _i18n/en/documentation/general/other-platforms.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+[Vocal](https://vocalproject.net/)|No|No|Yes
+[gPodder](https://gpodder.github.io/)|Yes|Yes|Yes
```
Comment (ByteHamster) [inline _i18n/en/documentation/general/other-platforms.md:8]: Gpodder runs on iOS?

File: _i18n/en/documentation/general/other-platforms.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+However attractive the title may sound, AntennaPod is not and will not be available on Windows, iOS or the Linux desktop. The app is made by a small team of volunteers, and making the app for these other platforms would be a lot of work.
+
+There are (open source) alternatives, however, which we would like to point you to:
+App/Application|Windows|iOS|Linux
+---|---|---|---
```
Comment (ByteHamster) [inline _i18n/en/documentation/general/other-platforms.md:5]: Maybe sort columns to have mobile iOS client in the front, not between desktops? Add MacOS?

File: _i18n/en/documentation/general/other-platforms.md (hunk 2 above)
Comment (keunes) [reply _i18n/en/documentation/general/other-platforms.md:10]: I wouldn't do that here; it's not the focus/point of this article. And once Open Podcast API is implemented, I'd rather point to a list that is maintained on that website (so projects don't have to duplicate work).

File: _i18n/en/documentation/general/other-platforms.md (hunk 7 above)
Comment (keunes) [reply _i18n/en/documentation/general/other-platforms.md:8]: Hmm. Good point. Maybe we should split them up.

File: _i18n/en/documentation/general/f-droid.md (hunk 3 above)
Comment (keunes) [reply _i18n/en/documentation/general/f-droid.md:5]: When writing the text I was hesitating to include what you wrote. I then decided against it because it adds a level of detail (extra days between first Google upload to small percentage of users and GitHub tag) that is not relevant for the F-Droid users.
Would you be OK with leaving as is, not to make the text longer (I already feel I've been too wordy).

Comment (keunes) [reply _i18n/en/documentation/general/f-droid.md:5]: Hmm. Not sure adding this would've prevented the user from creating a ticket. Arguably they didn't even look at our documentation. Also, "When the new version is ready for the public" and "when we are sure that there are no critical bugs" are a bit contradictory: why do we see something as 'ready for the public' if we're not sure yet that there are no critical bugs...

File: _i18n/en/documentation/general/other-platforms.md (hunk 1 above)
Comment (keunes) [reply _i18n/en/documentation/general/other-platforms.md:9]: It's one of the only two open source clients for iOS and I have "open source" in brackets. I would prefer not make any value judgement in addition to set out criteria.

File: _i18n/en/documentation/general/f-droid.md (hunk 3 above)
Comment (ByteHamster) [reply _i18n/en/documentation/general/f-droid.md:5]: On Google Play, we are a lot more flexible (can stop roll-out, quickly upload an update, etc). So being extra sure that there are no critical bugs does make sense, even if it is already ready for the public. I do have to admit that it sounds a bit weird here, though. Not sure how to best say this.

File: _i18n/en/documentation/general/other-platforms.md (hunk 6 above)
Comment (keunes) [reply _i18n/en/documentation/general/other-platforms.md:6]: I've added a cross for 'no' because with all the empty spaces the table became hard to read. If we have better table formatting in future, we can remove the crosses.

Comment (ByteHamster) [reply _i18n/en/documentation/general/other-platforms.md]: hmm, but with crosses, we have the same problem as before - completely filled and hard to see at a glance which is which

Comment (keunes) [reply _i18n/en/documentation/general/other-platforms.md]: > How about centering the symbols below the table heading?

That'd be nice indeed. But that's also [table styling](https://www.codeply.com/p/O8BLz8ic1X) that can't be done in markdown. So I'll save that for later.

Comment (ByteHamster) [reply _i18n/en/documentation/general/other-platforms.md]: Markdown does support centering table content

Comment (keunes) [reply _i18n/en/documentation/general/other-platforms.md]: Sorry, I searched but apparently not properly. Will update it.
